coc/clans.py

```diff
@@ -48,11 +48,9 @@ class RankedClan(BaseClan):
     member_count: :class:`int`
         The number of members in the clan.
     points: :class:`int`
-        The clan's trophy-count. If retrieving info for capital or versus leader-boards, this will be ``None``.
+        The clan's trophy-count. If retrieving info for versus leader-boards, this will be ``None``.
     versus_points: :class:`int`
-        The clan's versus trophy count. If retrieving info for regular or capital leader boards, this will be ``None``.
-    capital_points: :class:`int`
-        The clan's capital trophy count. If retrieving info for regular or versus leader boards, this will be ``None``.
+        The clan's versus trophy count. If retrieving info for regular leader boards, this will be ``None``.
```

Comment:
These changes are there because your branch is based on a version without capital leaderboards. Please merge them in.